packages/bundler/src/url-utils.ts

```diff
@@ -28,8 +28,9 @@ import constants from './constants';
  */
 export function appendUrlPath(url_: string, extension: string): string {
   const uri = Uri.parse(url_);
-  uri['_path'] = `${uri.path}${extension}`;
-  return uri.toString();
+  const newUri =
+      Uri.from(Object.assign(uri, {path: `${uri.path}${extension}`}));
```

Comment:
nit 1: prefer object spread over `Object.assign`
nit 2: can't this be `uri.path += extenstion;`?
nit 3: `url_` can be renamed to remove the `_`

I don't think any of these need to be fixed if the PR is ready to merge as is.

Reply:
Re nit 1: I couldn't do object spread because uri has more properties than are accepted by the type constraint for Uri.from(x) and got around it with loose Object.assign typing.  But that's bad so I am explicit now with properties.  Addressed the other nits.  See updated code:

```ts
export function appendUrlPath(url: string, extension: string): string {
  const {scheme, authority, path, query, fragment} = Uri.parse(url);
  return Uri.from({scheme, authority, path: path + extension, query, fragment})
      .toString();
 }
```